ray-on-gke/user/kuberay-tpu-webhook/deployments/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kuberay-tpu-webhook
+  name: kuberay-tpu-webhook
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kuberay-tpu-webhook
+  template:
+    metadata:
+      labels:
+        app: kuberay-tpu-webhook
+    spec:
+      containers:
+        - image: ryanaoleary/kuberay-tpu-webhook:latest
```

Comment:
We should be pushing an official image here. Can we use Louhi to push an image to gcr.io?

Reply:
I synced with Han and he said he can handle the Louhi automation for the TPU webhook. In the meantime I went ahead and changed the Dockerfile to use the internal golang image and will see if I can publish the image to Artifact Registry.